packs/hpe-icsp/actions/icsp_buildplan_apply.yaml

```diff
@@ -0,0 +1,28 @@
+---
+  name: "icsp_buildplan_apply"
+  runner_type: "run-python"
+  description: "Apply Buidlplan to Specified Servers"
+  enabled: true
+  entry_point: "icsp_buildplan_apply.py"
+  parameters:
+    buildplan_uris:
+      type: "array"
+      description: "Comma seperated list of build plan URIs"
+      required: true
+      position: 1
```

Comment:
`position` is only used for local runner script actions, but it also doesn't break anything if you leave it here (in the future we might use it for sort order or similar :)).